app/models/user.rb

```diff
@@ -175,6 +177,14 @@ def ghost?
     id == GHOST_USER_ID
   end
 
+  def hide_introducer(name)
+    introducers.create_or_find_by!(name: name)
+  end
+
+  def introducer_hidden?(name)
+    introducers.where(name: name).exists?
+  end
```

Comment:
I made these methods on the `User` model. I could also make them commands, but I felt that it was nice to be able to do `current_user&.introducer_hidden?('scratchpad')`, but I have no idea if this is something we want.